_includes/nav.html

```diff
@@ -64,7 +64,7 @@
         {%- if node.has_children -%}
           <a href="#" class="nav-list-expander"><svg viewBox="0 0 24 24"><use xlink:href="#svg-arrow-right"></use></svg></a>
         {%- endif -%}
-        <a href="{{ node.url | absolute_url }}" class="nav-list-link{% if page.url == node.url %} active{% endif %}">{{ node.title }}</a>
+        <a href="/snowiq-docs{{ node.url }}" class="nav-list-link{% if page.url == node.url %} active{% endif %}">{{ node.title }}</a>
```

Comment:
@DanHealey  I'm pretty sure this will not work on our production website, for example https://docs.snowiq.net/soc-user-management.html

Reply:
@DanHealey if I see this correct, this is the change you made yesterday in response to the issue we had with the relative links not being generated correctly. The links are now working again in my fork, and I think that everything should be fine without the change.

Reply:
Yes, the change was made because of the issue with relative links. I will change this.